.travis.yml

```diff
@@ -10,21 +10,25 @@ env:
     - VARNISH_VERSION=5.1
     - SYMFONY_PHPUNIT_VERSION=8
     - SYMFONY_DEPRECATIONS_HELPER="max[self]=0"
-    - DEPENDENCIES="toflar/psr6-symfony-http-cache-store:^2.2.0"
+    - DEPENDENCIES="toflar/psr6-symfony-http-cache-store:^2.2.0\|^3.0.0"
 
 matrix:
   fast_finish: true
   include:
       # Minimum supported versions
-    - php: 7.1
+    - php: 7.2
       env: VARNISH_VERSION=3.0 SYMFONY_PHPUNIT_VERSION=5.7 COMPOSER_FLAGS="--prefer-lowest" DEPENDENCIES=""
 
-    - php: 7.1
-      env: SYMFONY_PHPUNIT_VERSION=7
     - php: 7.2
     - php: 7.3
     - php: 7.4
       env: DOCCHECK=true COVERAGE=true PHPUNIT_FLAGS="-v --coverage-text" VARNISH_MODULES_VERSION=0.12.1
+      python: 3.6
```

Comment:
lets drop this line as it did not help. we can ignore the failure and i will comment out the doccheck in a separate PR

Reply:
Done in a05cb960adf73894e936e746413eb6e890b69cf9